action/protocol/execution/evm/evm.go

```diff
@@ -144,10 +144,11 @@ func newParams(
 		Transfer:    MakeTransfer,
 		GetHash:     getHashFn,
 		Coinbase:    common.BytesToAddress(blkCtx.Producer.Bytes()),
+		GasLimit:    gasLimit,
 		BlockNumber: new(big.Int).SetUint64(blkCtx.BlockHeight),
 		Time:        new(big.Int).SetInt64(blkCtx.BlockTimeStamp.Unix()),
 		Difficulty:  new(big.Int).SetUint64(uint64(50)),
-		GasLimit:    gasLimit,
+		BaseFee:     new(big.Int).SetUint64(1), // in units of 10^-6 IOTX, current basefee is 1
```

Comment:
how is `BaseFee` used?

Reply:
see https://eips.ethereum.org/EIPS/eip-3198

it is for contract to access the basefee

Reply:
ok. Why is it set to `1`?